Source/JavaScriptCore/wasm/WasmFunctionParser.h

```diff
@@ -1096,6 +1173,59 @@ FOR_EACH_WASM_MEMORY_STORE_OP(CREATE_CASE)
             m_expressionStack.constructAndAppend(Types::I32, result);
             return { };
         }
+        case GCOpType::StructNew: {
+            uint32_t typeIndex;
```

Comment:
check if wasm gc is enabled?

Reply:
it isn't needed because we have a check for GCPrefix:
```
case GCPrefix: {
        WASM_PARSER_FAIL_IF(!Options::useWebAssemblyGC(), "Wasm GC is not enabled");
```